ajax/libs/labelauty/1.1.0/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "jquery-labelauty",
+  "version": "1.1.0",
+  "author": "Francisco Neves <contact@francisconeves.com>",
+  "repository": {
+    "type":"git",
+    "url":"git://github.com/fntneves/jquery-labelauty"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fntneves/jquery-labelauty",
+    "fileMap": [
+      {
+        "basePath": "source",
+        "files": [
+         "jquery-labelauty*",
+         "images/*.png"
+        ]
+      }
+    ]
+  },
+  "description": "A lightweight and beautiful jQuery plugin for radio and checkbox inputs.",
+  "filename": "source/jquery-labelauty.js",
```

Comment:
@menuka94 sorry that I typed it wrong, I mean filename should not be "source/jquery-labelauty.js"